libs/api_common/windows_platform_common.cpp

```diff
@@ -685,9 +708,15 @@ _load_all_program_data_information()
     return result;
 }
 
-_Must_inspect_result_ ebpf_result_t
-load_ebpf_provider_data()
+static void
```

Comment:
Suggest adding a // comment in the code above this function explaining why not to call it automatically at DLL time.